src/meta/processors/usersMan/AuthenticationProcessor.cpp

```diff
@@ -198,19 +198,20 @@ void ListUsersProcessor::process(const cpp2::ListUsersReq& req) {
 
 
 void ListRolesProcessor::process(const cpp2::ListRolesReq& req) {
-    auto spaceRet = getSpaceId(req.get_space());
-    if (!spaceRet.ok()) {
-        LOG(ERROR) << "space dose not found";
-        handleErrorCode(MetaCommon::to(spaceRet.status()));
-        onFinished();
-        return;
+    auto spaceId = req.get_space_id();
+    /**
+     * Because the god role's space id is kDefaultSpaceId.
```

Comment:
I am confused about this changes. 
"show roles" must be invoked under some space, right?  So why use defaultSpaceId here.

Reply:
> I am confused about this changes.
> "show roles" must be invoked under some space, right? So why use defaultSpaceId here.

Good point.
Actual the "show roles" must be invoked under some space. show the use the defaultSpaceId  is redundant.
let me correct .